### ISSUE TITLE
add createBond / Bonding status 

### ISSUE THREAD

**Issue by FaroukBoussarsar** [PhilipsHue/flutter_reactive_ble#628]
when connect to device using android double pairing request is received.
so as a work around if it is possible to bond from the side of the app it will solve this issue

https://developer.android.com/reference/android/bluetooth/BluetoothDevice#createBond()

this feature also helps detect if bounding is done or not so it minimise handling from our side

**Comment by vincent-iQontrol** [comment]
Indeed, it would be nice to implement some sort of bonding helper: https://gist.github.com/dglozano/9b0ce38a558eeca16137909bd368698c
We currently facing problems on android devices with "just works" implementation when all services are hidden before bonding.
